### DISCUSSION TITLE
Auto3DSeg "region based training" / hierarchical label

### DISCUSSION THREAD

**Discussion by mibaumgartner** [Project-MONAI/MONAI#7436] [General]
Hey, I would like to train Auto3DSeg with "region based labels" (at least this is how it is called in nnU-Net :) ), similar to how BRATS training is usually performed (where the labels are hierarchical). Is this possible with Auto3DSeg? 

Best,
Michael

**Comment by KumoLiu** [comment]
Hi @mibaumgartner, I think yes, you can refer to this tutorial for more details.
https://github.com/Project-MONAI/tutorials/pull/1627/files
Thanks.

**Comment by mibaumgartner** [comment]
Thanks for the quick answer, this is exactly what I was looking for :)
